package.json

```diff
@@ -4,21 +4,26 @@
   "private": false,
   "dependencies": {
     "@apollo/client": "^3.2.9",
+    "@craco/craco": "^6.0.0",
+    "@tailwindcss/postcss7-compat": "^2.0.2",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "autoprefixer": "^9",
     "graphql": "^15.4.0",
     "graphql-query-builder": "^1.0.7",
-    "react": "^16.13.0",
-    "react-dom": "^16.13.0",
-    "react-scripts": "3.4.3",
+    "postcss": "^7",
+    "react": "^17.0.1",
```

Comment:
@steveoni again, doesn't it break Storybook? If you remember we had similar discussion here - https://github.com/datopian/data-explorer-graphql/pull/7#discussion_r534675362

Reply:
oh. not aware of that. I will change that